profile-widget/profilewidget2.cpp

```diff
@@ -828,6 +828,8 @@ void ProfileWidget2::plotDive(struct dive *d, bool force, bool doClearPictures)
 		clearPictures();
 	else
 		plotPictures();
+
+	toolTipItem->refresh(mousePos);
```

Comment:
Perhaps instead of saving mousePos perhaps
```
	QPoint pos = mapFromGlobal(QCursor::pos());
	toolTipItem->refresh(pos);
```
?
Matter of fact - if I quickly move the mouse cursor out of the profile, with the current code I get random values [where the cursor was last registered on the widget] on key-right,left.

Reply:
Agree. The storing of mousePos is the part I do not like as well. That said ... it should be something like `mapToScene(mapFromGlobal(QCursor::pos()))` but that gives a small error compared to `mapToScene(event->pos())` small as in 2 pixels. That results in a small jump of the data in the info box when pressing left/right. Let me see if I can understand where that small difference is coming from.

Reply:
force pushed a new version.